src/IROperator.h

```diff
@@ -678,6 +678,22 @@ inline Expr max(int a, const Expr &b) {
     return Internal::Max::make(Internal::make_const(b.type(), a), b);
 }
 
+/** Returns an expression representing the greater of an expressions
+ * list, after doing any necessary type coersion using
+ * \ref Internal::match_types. Vectorizes cleanly on most platforms
+ * (with the exception of integer types on x86 without SSE4).
+ * The expressions are folded from right ie. max(.., max(.., ..)). */
```

Comment:
comment lies about fold type
It's arbitrary decision for me since it's an associative operator..
Fixed comments locally, but might spend some time on how simplification looks.